### DISCUSSION TITLE
Stuck on "Setting up your codespaces" - Seeking Help

### DISCUSSION THREAD

**Discussion by yashleo** [community/community#127683] [Codespaces]
### Select Topic Area

Bug

### Body

Hi everyone,

I'm having trouble getting my Codespace to launch. It keeps getting stuck on the "Setting up your codespaces" message and never progresses further. I've been trying for last 2 months and have made several attempts, but no luck.

**Network Errors (all with error code ERR_NAME_NOT_RESOLVED):**

- assets.github.dev/static/splash-screen-styles.css
- app.3abbf6cfce04d31517f6.js
- Many files starting with "commons-app~bootstrap~pfHelper-" (e.g., commons-app~bootstrap~pfHelper-bn.js)
- Many files starting with "commons-app~" (e.g., commons-app-index.js)
- Many other files with various names (e.g., 371.a061e5ce422731d05401.js)

**Console Warnings:**
![setting up your codespaces](https://github.com/community/community/assets/103138690/8e347382-a774-4c61-be54-40426b394d43)

The resource [https://assets.github.dev/static/app.3abbf6cfce04d31517f6.js] was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate as value and it is preloaded intentionally. (This warning appears twice)

**Troubleshooting Steps Taken:**

- Refreshing the page
- Restarting your browser
- Trying a different browser
- Checking your internet connection

**Here's some additional info that might be helpful:**

- I'm using Windows 11 
- I'm using Google Chrome 
- It is not working on any repository
- I have successfully created CodeSpaces before

### **I'm hoping someone in the community might have some ideas on how to fix this.  Thanks in advance for your help!**

**Comment by lesampietro** [comment]
I am having a similar issue for 2 days now. Please community, help us :(

**Comment by lesampietro** [comment]
I have fixed the problem by using the solution in the below post
https://github.com/orgs/community/discussions/58374#discussioncomment-6253155

**Reply by yashleo** [reply]
It was so nice of you to suggest it. Actually, it was working when i used vpn, but that makes it really slow to load and difficult to use. But I found this is not because of GitHub, maybe this is because of your internet service provider (ISP). If it's local, you can try contacting them!

**Reply by lesampietro** [reply]
I did as they said in the link post: turned on the vpn, reloaded the codespace, and then turned the vpn back off. It is now working normally again. But maybe my problem is different than yours :/
